documentation.md

```diff
@@ -1,7 +1,6 @@
 ---
 layout: page
-title: Docs
-external-link: //pelux.io/software-factory
+title: Documentation
```

Comment:
I would propose not rename Docs, or to use the term "Handbook" instead. Perhaps check with Andreas? Note that we will add a section for Architecture soon and then we likely need to change the term again.

Reply:
I agree with calling it Handbook instead. I have felt that "Documentation" has felt too general.